capella2polarion/converters/element_converter.py

```diff
@@ -441,6 +479,30 @@ def __generic_work_item(
 
         return converter_data.work_item
 
+    def _add_attributes(
+        self,
+        converter_data: data_session.ConverterData,
+        attributes: list[dict[str, t.Any]],
+    ):
+        assert converter_data.work_item is not None
+        for attribute in attributes:
+            try:
+                value = _resolve_capella_attribute(converter_data, attribute)
+                setattr(
+                    converter_data.work_item, attribute["polarion_id"], value
+                )
```

Comment:
we could also use the `additional_attributes` dict here to set the value like we do it in other serializers. This should enhance readability